server/protocols/jmap-integration-testing/memory-jmap-integration-testing/pom.xml

```diff
@@ -161,17 +161,4 @@
         </dependency>
     </dependencies>
 
-    <build>
```

Comment:
Why? `12 min -> 4 min` is nice... Why would we abandon that?

Reply:
Because:
1/ we don't need this anymore -> we are filtering tests by product, so we will play less tests.
  for example a few for Cassandra product
2/ having a maven dependency in order to play some tests before other is a ugly trick, IMO

Reply:
You did not get it.

Here the surefire configuration you removed is about fasting up JMAP memory integration tests. It's not connected to (1/) as it is scopped in memory where we keep playing all tests. And definitly it is not an ugly ordering trick!

The plugin you removed was about playing JMAP memory integration tests in parrallel significantly fasting them up.

I will re-add this.

(niote that I 100% agree to remove the dependency ordering trick and did not even commented on it.